torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4023,6 +4023,17 @@ def _log_create_unbacked_symbol(
             maybe_extra_debug,
             stack_info=is_debug,
         )
+        trace_structured(
+            "create_unbacked_symbol",
```

Comment:
is it still the case that each of these new string names ("create_unbacked_symbol") requires patches to tlparse to get working? (and `trace_structured("artifact", ...)` is the blessed way to add new artifacts without needing tlparse changes)

Reply:
Yes. I intentionally don't want artifacts for these because you will completely flood the artifact list (you're talking O(100) events here